src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/InstallAppsTask.groovy

```diff
@@ -73,6 +114,18 @@ class InstallAppsTask extends AbstractTask {
         return archiveName;
     }
     
+    private String getInstallAppPackages() {
+        if (project.plugins.hasPlugin("ear")) {
+            project.liberty.installapps.installAppPackages = "project"
+        }
```

Comment:
ok - that's fine.  I keep wondering if we really need installAppPackages as a flag.  Something to think more about when we add the install dependencies code.  If the project is an ear or war, we install the result.  If there are dependency applications defined, we install those too.